Makefile

```diff
@@ -154,10 +154,10 @@ ifdef use
   endif
 endif
 
-ifeq ($(strip $(usedebugger)),lldb)
-  debuggercmd := lldb --batch --one-line "breakpoint set --name main" --one-line run --one-line "process handle SIGINT --pass true --stop false" --one-line "process handle SIGUSR2 --pass true --stop false"  --one-line "thread continue" --one-line-on-crash "frame variable" --one-line-on-crash "register read" --one-line-on-crash "bt all" --one-line-on-crash "quit 1" --
-else ifeq ($(strip $(usedebugger)),gdb)
-  debuggercmd := gdb --quiet --batch --return-child-result --eval-command="set confirm off" --eval-command="set pagination off" --eval-command="handle SIGINT nostop pass" --eval-command="handle SIGUSR2 nostop pass" --eval-command=run  --eval-command="info args" --eval-command="info locals" --eval-command="info registers" --eval-command="thread apply all bt full" --eval-command=quit --args
+ifneq ($(findstring lldb,$(usedebugger)),)
```

Comment:
so what does this change do?

Reply:
It makes it possible to use a path (that contains the string "lldb") to explicitly set which lldb you want to use.